Templates/Integrations/DataEnrichmentThreatIntelligence/DataEnrichmentThreatIntelligence.py

```diff
@@ -0,0 +1,320 @@
+
+        Returns:
+            Response JSON
+        """
+        suffix = 'analysis'
+        params = {'hash': file_hash}
+        return self._http_request('GET', suffix, params=params)
+
+    def get_domain_request(self, domain):
+        """Building request for file command
+
+        Args:
+            domain: Domain to search in API
+
+        Returns:
+            Response JSON
+        """
+        suffix = 'analysis'
+        params = {'domain': domain}
+        return self._http_request('GET', suffix, params=params)
+
+
+def build_context(results: Union[Dict, List], indicator_type: str) -> Union[Dict, List]:
+    """Formatting results from API to Demisto Context
+
+    Args:
+        results: raw results from raw_response
+        indicator_type: type of indicator
+
+    Returns:
+        Results formatted to Demisto Context
+    """
+
+    def build_entry_context(entry: Dict):
+        return {
+            'ID': entry.get('id'),
+            'Severity': entry.get('severity'),
+            indicator_type: entry.get('indicator'),
+            'Description': entry.get('description')
+        }
+
+    if isinstance(results, list):
+        return [build_entry_context(entry) for entry in results]  # pragma: no cover
+    return build_entry_context(results)
+
+
+''' COMMANDS '''
+
+
+def search_ip(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Gets results for the API.
+    """
+    ip = args.get('ip', '')
+    raw_response = client.get_ip_request(ip)
+    results = raw_response.get('result')
+    if results:
+        result = results[0]
+        title = f'{INTEGRATION_NAME} - Analysis results for IP: {ip}'
+        context_entry = build_context(result, 'IP')
+        # Building a score for DBot
+        score = client.calculate_dbot_score(result.get('severity'))
+        dbot_entry = build_dbot_entry(ip, 'ip', INTEGRATION_NAME, score, result.get('description'))
+        context = {
+            f'{INTEGRATION_CONTEXT_NAME}(val.ID && val.ID === obj.ID)': context_entry
+        }
+        context.update(dbot_entry)
+        human_readable: str = tableToMarkdown(title, context_entry, removeNull=True)
+        return human_readable, context, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - No results found for IP: {ip}', {}, raw_response
+
+
+def search_url(client: Client, args: Dict) -> Tuple[str, Dict, Dict]:
+    """Gets a job from the API. Used mostly for polling playbook
+    """
+    url = args.get('url', '')
+    raw_response = client.get_url_request(url)
+    results = raw_response.get('result')
+    if results:
+        result = results[0]
+        title = f'{INTEGRATION_NAME} - Analysis results for URL: {url}'
+        context_entry = build_context(result, 'URL')
+        # Building a score for DBot
+        score = client.calculate_dbot_score(result.get('severity'))
+        dbot_entry = build_dbot_entry(url, 'url', INTEGRATION_NAME, score, result.get('description'))
+        context = {
```

Comment:
need also to return here according to context standard

Reply:
`build_dbot_entry` already doing so